test/e2e/storage/persistent_volumes-local.go

```diff
@@ -307,6 +310,44 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("should be able to mount volume in two pods at the same time and NOT write from pod1, as pod2 overwritten volume's SELinux label", func() {
+		It("should be able to write from pod1 and read from pod2", func() {
```

Comment:
"a second pod using a different label should not be able to read or write to the same volume"